pkg/azureutils/azure_disk_constants.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package azureutils
+
+import (
+	"regexp"
+)
+
+const (
```

Comment:
I wonder if we should move these to their own `constants` package to consistently access them through that alias rather than both `constants` and `azureutils`?

Reply:
I was also thinking of making constants separate package but left it as this for now. If there are no objections for having them as separate package, I would love to make that change.

Reply:
I have no objects - I believe it would be a good thing to do. @andyzhangx What do you think?

Reply:
use `consts` instead of `constants` ?

Reply:
I have refactored constants into a separate pkg.